src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/X86/Sse2.cs

```diff
@@ -1313,7 +1313,7 @@ public static unsafe Vector128<double> SetVector128(double e1, double e0)
         /// __m128i _mm_sad_epu8 (__m128i a,  __m128i b)
         ///   PSADBW xmm, xmm/m128
         /// </summary>
-        public static Vector128<long> SumAbsoluteDifferences(Vector128<byte> left, Vector128<byte> right) => SumAbsoluteDifferences(left, right);
```

Comment:
Shouldn't this be `Vector128<ushort>`?

> Computes the absolute differences of the
packed unsigned byte integers from xmm2
/m128 and xmm1; the 8 low differences and 8
high differences are then summed separately
to produce two unsigned word integer results.

Reply:
Ah, right! Will fix. 
Thank you!

Reply:
Done. Now they are correct and consistent with `MultipleSumAbsoluteDifferences`.